plutus-core/plutus-core/src/PlutusCore/DeBruijn/Internal.hs

```diff
@@ -196,15 +202,19 @@ getIndex u = do
         -- and thus the computation 'current-foundLvl' is '>=0' and its conversion to Natural will not lead to arithmetic underflow.
         fromIntegral $ current - foundLvl
 
+    -- Because of invariant-B, the levelMapping contains only positive levels;
```

Comment:
Is this meant to be here?

Reply:
I think it could still be below, it just explains the cases in which the handler will be called.